### PR TITLE
[#736] Use binaries list from octez and partially automate binaries update

### PR DESCRIPTION
**Pull request by krendelhoff2**
## Related issue(s)

Resolves #

#### Related changes (conditional)

- [ ] I checked whether I should update the [README](/serokell/tezos-packaging/tree/master/README.md)

- [ ] I checked whether native packaging works, i.e. native binary packages
  can be successfully built.

#### Stylistic guide (mandatory)

- [ ] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).

### CODE REVIEW

File: scripts/shell.nix [hunk 1]
```diff
@@ -8,6 +8,7 @@ mkShell {
     # NOTE using the most recent nix for sticky nix flake lock update feature
     nix
     python3
+    moreutils
```
Comment (DMozhevitin) [inline scripts/shell.nix:11]: Do we actually use the tools from `moreutils` in this PR? 🤔

**Review by DMozhevitin — APPROVED** [review]
Looks good 👍 , have only one question

Btw, is it possible to make a dummy tag like [this one](https://github.com/serokell/tezos-packaging/releases/tag/test-736) to check how it works in action?

File: scripts/shell.nix (hunk 1 above)
Comment (krendelhoff2) [reply scripts/shell.nix:11]: yes, `proto` script requires `sponge` utility from `moreutils`

Comment (DMozhevitin) [reply scripts/shell.nix:11]: Got it 👌

**Pull request merged**